cuenca_validations/types/requests.py

```
@@ -480,6 +487,13 @@ def validate_birth_date(cls, curp: Optional[Curp]) -> Optional[Curp]:
             validate_age_requirement(curp)
         return curp
 
+    @field_validator('profession')
+    @classmethod
+    def validate_profession(cls, profession: Profession) -> Profession:
+        if profession == Profession.otro:
+            raise ValueError('Profession "otro" is not allowed for this model')
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)